packages/expo-screen-orientation/ios/EXScreenOrientation/EXScreenOrientationRegistry.m

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019-present 650 Industries. All rights reserved.
+
+#import <EXScreenOrientation/EXScreenOrientationRegistry.h>
+#import <UMCore/UMDefines.h>
+
+@implementation EXScreenOrientationRegistry
```

Comment:
I don't think at the moment `expo-screen-orientation` needs something like `appId`-recognizing registry — there is no notion of `appId`s in unimodules yet and they aren't compatible with multi-instance native apps, so let's make this singleton a very easy registry that will remember mask, not a dictionary of masks and move the scoping to Expo client codebase where `experienceId` is a thing

Reply:
This will also ensure that we don't use singleton modules illegally — via `class`, singleton modules should be referenced by name `getSingletonModuleForName:@"ScreenOrientationManager"`

Reply:
As per the live discussion we agreed to leave this logic here in preparation for adding support for identifiers to modules